addons/source-python/packages/source-python/players/_base.py

```diff
@@ -262,7 +274,7 @@ def set_team(self, value):
 
     team = property(get_team, set_team)
 
-    @property
+    @cached_property
```

Comment:
Theoretically, the client's language can be changed. So, I'm not sure if we should cache it.

Reply:
True, but since it is already cached [here](https://github.com/Source-Python-Dev-Team/Source.Python/blob/master/addons/source-python/packages/source-python/players/_language/cache.py) I think it is overall beneficial to only query that external dictionary once and bind the result.